e2e-tests/endpoints/kusama/blocks/2786879.json

```diff
@@ -148,7 +148,7 @@
             "info": {
                 "weight": "195000000",
                 "class": "Normal",
-                "partialFee": "3333333"
+                "partialFee": "166666666"
```

Comment:
why is this partialFee changed?

I guess because the historialApi is used or the calculate fee bug....

Reply:
Yuo so I found a bug for calculating fees in regards to historic blocks. I fixed it in this PR thats why I changed the e2e tests too for partialFee responses.